### ISSUE TITLE
Add accessibility support (component is not focusable via TalkBack)

### ISSUE THREAD

**Issue by yannick-softwerft** [fateh999/react-native-paper-dropdown#135]
This component is not usable with assistive technology like talkback. Could that be supported?
